.gitignore

```diff
@@ -13,4 +14,15 @@ pids
 .idea
 .vscode
 yarn.lock
-package-lock.json
+tmp
+jsconfig.json
+types
+docs/api
+docs/typedefs
+.history
+.docusaurus
+tsconfig.tsbuildinfo
+apify_storage
+crawlee_storage
+storage
+.turbo
```

Comment:
i dont think we want to pollute the root gitignore file, this should go to the website folder, no?

Reply:
Huh, some names do not even sound like anything from docs / apify-client-js AFAIK, it looks like copy-pasted from a different repo? but agreed, moving the changes to `website` shouldn't do any harm